load.php

```diff
@@ -330,12 +330,12 @@ function perflab_get_standalone_plugin_version_constants( $source = 'plugins' )
 	 * as `$plugin_slug => $version_constant` pairs.
 	 */
 	return array(
-		'webp-uploads'               => 'WEBP_UPLOADS_VERSION',
-		'dominant-color-images'      => 'DOMINANT_COLOR_IMAGES_VERSION',
-		'image-loading-optimization' => 'IMAGE_LOADING_OPTIMIZATION_VERSION',
-		'performant-translations'    => 'PERFORMANT_TRANSLATIONS_VERSION',
-		'auto-sizes'                 => 'IMAGE_AUTO_SIZES_VERSION',
-		'speculation-rules'          => 'SPECULATION_RULES_VERSION',
+		'webp-uploads'            => 'WEBP_UPLOADS_VERSION',
+		'dominant-color-images'   => 'DOMINANT_COLOR_IMAGES_VERSION',
+		'optimization-detective'  => 'OPTIMIZATION_DETECTIVE_VERSION',
```

Comment:
Per the #1046 discussion, let's remove this from here for now, as having it here will lead it to show up in the PL recommended features. The only other purpose for this function was to check for the constant to avoid conflict between a plugin and its equivalent module, which is no longer relevant as of #1030, and in case of Optimization Detective wouldn't be relevant anyway since it's always only going to have been a plugin.

Maybe instead put a TODO comment here to add it once deemed stable enough / split into image loading optimization feature plugin.

Reply:
Like so?

```suggestion
		// TODO: Add image optimization dependent plugin of Optimization Detective once ready for end users.
```

Reply:
SGTM!